### ISSUE TITLE
Errors when testing

### ISSUE THREAD

**Issue by lqh-dlut** [fudan-zvg/DeMo#14]
Thanks for your excellent work!
Unfortunately, when I run the training code, I get some errors:

![Image](https://github.com/user-attachments/assets/8ed22588-a990-4a1f-8bc9-a03001e478e7)

The only thing I changed was to set limit_train_batches, limit_val_batches and max_epochs with specific and small numbers in the train.py file (for simple testing). I'm not sure if this had any effect on the final test? 

There is nothing wrong with the training and validation phases.

Could you help me to solve this issue? Thanks!

**Comment by Zbozhou** [comment]
Sorry, I have not encountered this issue before, but based on your information, there seems to be an issue in this [line](https://github.com/fudan-zvg/DeMo/blob/main/src/utils/submission_av2.py#L65). Could you check if there is something wrong with `self.submission_file`?

**Comment by lqh-dlut** [comment]
Do you mean the .parquet file saved in submission folder? There are no files in my submission folder.

I notice there is another error before`Error executing job with overrides: ['gpus=1', 'test=true']`
If there is any relationship?

**Comment by Zbozhou** [comment]
This error is a standard report when an error occurs and is not related to your actual issue. Maybe you can try reinstalling AV2, but I cannot confirm the exact cause of the error.

**Comment by lqh-dlut** [comment]
Yes, you are correct! The AV2 was updated two months before (now is 0.3.1). The error disappear after reinstalling the 0.2.1 version. Thank you for your kind help.

I would also like to inquire if there is any way to directly display the test results, similar to the verification and trainning process, besides obtaining the Parquet file and uploading it to the leaderboard.

**Comment by Zbozhou** [comment]
Since the test set is secret, the test results can only be obtained by submitting the parquet file and uploading it to the leaderboard.

**Comment by lqh-dlut** [comment]
I see. Thanks a lot!
